app/doc/models.go

```diff
@@ -15,3 +17,26 @@ type userCreateTmpResponse struct {
 		ExpiresIn int32 `json:"expires_in"`
 	} `json:"data"`
 }
+
+// swagger:model groupsMembershipHistoryResponseRow
+type groupsMembershipHistoryResponseRow struct {
+	// `groups_groups.ID`
+	// required: true
+	ID int64 `json:"id"`
+	// `groups_groups.sStatusDate`
+	// required: true
+	StatusDate time.Time `json:"status_date"`
+	// `groups_groups.sType`
+	// required: true
+	// enum: invitationSent,requestSent,invitationAccepted,requestAccepted,invitationRefused,requestRefused,removed,left
+	Type string `json:"type"`
+
+	// required: true
+	Group struct {
```

Comment:
It does not seems to be really reusage in other context (as there are only name and type)...
Actually, where is it used?

Reply:
It is used only here (by the groupsMembershipHistory service)